### PR TITLE
Update altcoin accounts info popups

### PR DESCRIPTION
**Pull request by aejontargaryen**
reference issue https://github.com/bisq-network/bisq/issues/1885

as well as Arbitration instructions  +   version requirement for DRGL wallets. 

Asset reference PR https://github.com/bisq-network/bisq/pull/1762

### CODE REVIEW

**Review by ripcurlx — COMMENTED** [review]
utACK - as I never used the Monero wallet I can't say if the mentioned commands in the text are correct. @ManfredKarrer Do you have any concerns regarding the textual changes?

**Review by ManfredKarrer — COMMENTED** [review]
Did you check if those old commands on the terminal are still supported?

**Comment by aejontargaryen** [comment]
@ManfredKarrer yes same commands, just different cli-wallet name.
I included this additional link in the info box as well. 
https://www.getmonero.org/resources/user-guides/prove-payment.html

File: core/src/main/resources/i18n/displayStrings.properties [hunk 1]
```diff
@@ -1021,6 +1025,22 @@ Because Conceal is a privacy coin, block explorers cannot verify transfers.\n\n\
 If you cannot provide the required data to the arbitrator, you will lose the dispute case.\n\
 If you do not save the transaction secret key immediately after transferring CCX, it cannot be recovered later.\n\
 If you do not understand these requirements, seek help at the Conceal discord (http://discord.conceal.network).
+account.altcoin.popup.drgl.msg=Trading Dragonglass on Bisq requires that you understand and fulfill \
+the following requirements:\n\n\
+Dragonglass arms you with the power to prove payments while remaining anonymous thru use of your TXN-Private-Key.\n\
```
Comment (ManfredKarrer) [inline core/src/main/resources/i18n/displayStrings.properties]: I would prefer a more neutral language here. We are not promoting any coin (even it sounds interesting one). I will chang it myself to not postpone the merge...

**Review by ManfredKarrer — APPROVED** [review]
utACK

File: core/src/main/resources/i18n/displayStrings.properties (hunk 1 above)
Comment (aejontargaryen) [reply core/src/main/resources/i18n/displayStrings.properties]: @ManfredKarrer yes sir. thats best, as i do have a tendency to get carried away. :smile: Thanks!

**Pull request merged**